### PR TITLE
Hide Difficult questions tab if the quiz is still a draft

### LINKED ISSUE #12431: EQM - Reports - Getting a 500 error when viewing the 'Difficult questions' for a not started quiz
## Observed behavior

When I go to Coach > Reports > Quizzes and I attempt to preview the 'Difficult questions' for a just created not started quiz I am getting a server error:

```
{
  "data": "\n<!doctype html>\n<html lang=\"en\">\n<head>\n  <title>Server Error (500)</title>\n</head>\n<body>\n  <h1>Server Error (500)</h1><p></p>\n</body>\n</html>\n",
  "status": 500,
  "statusText": "Internal Server Error",
  "headers": {
    "content-length": "145",
    "content-type": "text/html",
    "date": "Tue, 09 Jul 2024 13:39:38 GMT",
    "referrer-policy": "same-origin",
    "server": "Cheroot/unknown",
    "vary": "Cookie",
    "x-content-type-options": "nosniff",
    "x-frame-options": "DENY"
  },
  "config": {
    "transitional": {
      "silentJSONParsing": true,
      "forcedJSONParsing": true,
      "clarifyTimeoutError": false
    },
    "adapter": [
      "xhr",
      "http",
      "fetch"
    ],
    "transformRequest": [
      null
    ],
    "transformResponse": [
      null
    ],
    "timeout": 0,
    "xsrfCookieName": "kolibri_csrftoken",
    "xsrfHeaderName": "X-CSRFToken",
    "maxContentLength": -1,
    "maxBodyLength": -1,
    "env": {},
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "X-Requested-With": "XMLHttpRequest"
    },
    "paramsSerializer": {},
    "url": "/coach/api/quizdifficulties/26/",
    "params": {
      "classroom_id": "ded7bae236498996c848802a0a39540e"
    },
    "method": "get"
  },
  "request": {}
}
```

Video:

https://github.com/learningequality/kolibri/assets/79847249/80fff39a-5c7b-4369-9851-b34b858e6966

## Expected behavior

The user should see the difficult questions table with text: There are no questions

## Steps to reproduce the issue

1. Install the build from https://github.com/learningequality/kolibri/releases/tag/v0.17.0-beta0
2. Create a new quiz with sections and questions and save it.
3. While at Coach > Reports > Quizzes click on the title of a quiz and then select the 'Difficult questions' tab

## Usage Details
Windows 11, Ubuntu 22  - Chrome

### PR DESCRIPTION
**Pull request by AlexVelezLl**
## Summary

Hide Difficult questions tab if the exam is still a draft.

non-draft quiz:
image

draft quiz:
image

## References
Closes #12431.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by marcellamaki — COMMENTED** [review]
Code changes look good to me, I'll wait for @pcenov's manual QA approval though. Thanks, @AlexVelezLl !!

**Review by pcenov — APPROVED** [review]
Thanks @AlexVelezLl - no issues observed while manually testing, the tab is hidden now.

**Review by marcellamaki — APPROVED** [review]

**Pull request merged**
